src/migrations/20200608212125-lesao.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.addColumn('lesao', 'id_collection', Sequelize.BIGINT, 
+            {
+                allowNull: false,
+                references: { model: "collection", key: "id" },
+                onUpdate: "CASCADE",
+                onDelete: "RESTRICT"
+            }
+        )
+        .then(
+            ()=>{
+                return Promise.all(
+                    [
+                        queryInterface.sequelize.query(
```

Comment:
As I mentioned, you need to rename the files to avoid

```
select * from lesao limit 10;
+----+----------+------------------------------------------------------+---------------------+---------------------+---------------+-------+
| id | nome     | detalhes                                             | created_at          | updated_at          | id_collection | grade |
+----+----------+------------------------------------------------------+---------------------+---------------------+---------------+-------+
|  1 | Unkown   | Unkown                                               | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |             1 |     0 |
|  2 | Negative | Negative for intraepithelial lesion                  | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |             1 |     1 |
|  3 | ASC-US   | Atypical squamous cells of undetermined significance | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |             1 |     2 |
|  4 | LSIL     | Low grade squamous intraepithelial lesion            | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |             1 |     3 |
|  5 | ASC-H    | Atypical squamous cells cannot exclude HSIL          | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |             1 |     4 |
|  6 | HSIL     | High grade squamous intraepithelial lesion           | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |             1 |     5 |
|  7 | SCC      | Squamous cell carcinoma                              | 2020-01-01 00:00:00 | 2020-01-01 00:00:00 |          NULL |     6 |
+----+----------+------------------------------------------------------+---------------------+---------------------+---------------+-------+
```